load/InputFileset.py

```diff
@@ -73,21 +73,24 @@ def __init__(self, config, location, filesetId, filesetPath, damId, typeCode, bi
 		self.bibleId = bibleId
 		self.index = index
 		self.languageRecord = languageRecord
-		self.filesetPrefix = "%s/%s/%s" % (self.typeCode, self.bibleId, self.filesetId)
 		if self.typeCode == "text":
-			self.databasePath = "%s%s.db" % (config.directory_accepted, damId)
+			self.databasePath = "%s%s.db" % (config.directory_accepted, self.textFilesetId())
+			# if filesetId is of type text and it formatted as a stocknumber, we can replaced 2 peer "_".
+			self.filesetId = self.filesetId.replace("2", "_")
 		else:
 			self.databasePath = None
 		if self.typeCode == "text" and len(self.filesetId) < 10:
 			# BWF. if we encounter this error message, go upstream and change filesetId to 10 chars
 			print("*** !!! text fileset with less than 10 characters !!! ***")
-			self.csvFilename = "%s%s_%s_%s.csv" % (config.directory_accepted, self.typeCode, self.bibleId, damId[:7] + "_" + damId[8:])
+			self.csvFilename = "%s%s_%s_%s.csv" % (config.directory_accepted, self.typeCode, self.bibleId, self.textFilesetId())
```

Comment:
can you search through the code to ensure there are no more places where we do this string manipulation
damId[:7] + "_" + damId[8:]
We should only ever do this once, in the textFilesetId method

Reply:
@bradflood I have added a refactor to make sure that this logic just happens once. You can see it in the following lines:

https://github.com/faithcomesbyhearing/dbp-etl/pull/80/files#diff-943fbc0f60984b469ccd792a22ead44a9b9eaa1c4e9050b1433f5997c812d252R194

https://github.com/faithcomesbyhearing/dbp-etl/pull/80/files#diff-160bcec181f4365fc933594ec0b0356893dc185d2f4d9fc9e8a81f732eb3ef15R235

https://github.com/faithcomesbyhearing/dbp-etl/pull/80/files#diff-943fbc0f60984b469ccd792a22ead44a9b9eaa1c4e9050b1433f5997c812d252R422